### ISSUE TITLE
Text Encoder LoRA keys not loaded for ANIMA

### ISSUE THREAD

**Issue by nArn0** [Haoming02/sd-webui-forge-classic#1115]
Since Forge Neo is using ComfyUI LoRA loader, it suffers from the same issues.

I have added the needed information in the open ticket over at Comfy:

https://github.com/Comfy-Org/ComfyUI/issues/13747#issuecomment-4415970047

**Comment by Haoming02** [comment]
Just be a sane person and don't train the Text Encoder 🫠

**Comment by plularias** [comment]
It seems to be a loader key-mapping issue. 
Forge's runtime TE path for ANIMA ends up under `text_encoders.qwen3_06b.model.*`, but current ANIMA LoRAs do not always match that path.

Forge mounts the ANIMA text encoder under `qwen3_06b`:

https://github.com/Haoming02/sd-webui-forge-classic/blob/3cce3b6de8c1e016af9e3a767e4f8a5c27bdace7/backend/diffusion_engine/anima.py#L19

Qwen3 weights are first imported under `qwen3_06b.transformer.*`:

https://github.com/Haoming02/sd-webui-forge-classic/blob/3cce3b6de8c1e016af9e3a767e4f8a5c27bdace7/backend/loader.py#L687-L691

Then that prefix is stripped when the text encoder component is extracted:

https://github.com/Haoming02/sd-webui-forge-classic/blob/3cce3b6de8c1e016af9e3a767e4f8a5c27bdace7/backend/loader.py#L794-L795

https://github.com/Haoming02/sd-webui-forge-classic/blob/3cce3b6de8c1e016af9e3a767e4f8a5c27bdace7/backend/state_dict.py#L37-L40

Also, the ANIMA Qwen3 module stores its transformer as `self.model`, so the runtime state-dict path is `qwen3_06b.model.*`:

https://github.com/Haoming02/sd-webui-forge-classic/blob/3cce3b6de8c1e016af9e3a767e4f8a5c27bdace7/backend/nn/llm/llama.py#L528-L542

So Forge expects TE LoRA keys to look like:
`text_encoders.qwen3_06b.model.*`

But current ANIMA LoRAs do not always match this:
1. Raw ANIMA LoRAs use `lora_te_layers_*`
2. Kohya's official conversion script outputs `text_encoders.qwen3_06b.transformer.model.*` (note the extra `.transformer.`):

https://github.com/kohya-ss/sd-scripts/blob/502cc3fab2aa22c106580e2e05c4692cfde5e5ff/networks/convert_anima_lora_to_comfy.py#L14-L15

https://github.com/kohya-ss/sd-scripts/blob/502cc3fab2aa22c106580e2e05c4692cfde5e5ff/networks/convert_anima_lora_to_comfy.py#L84-L87

Since neither format matches Forge's expected runtime path, the entire TE branch gets skipped and logged as unmatched.

Until the loader is patched, you can fix the LoRA manually as a workaround:
- Run the [official ANIMA conversion script](https://github.com/kohya-ss/sd-scripts/blob/main/networks/convert_anima_lora_to_comfy.py)
- Run a quick script to rename the TE prefix inside the converted .safetensors file:
```
text_encoders.qwen3_06b.transformer.model.
-> text_encoders.qwen3_06b.model.
```
Once renamed, the keys will align with Forge's runtime map.

suggested fix: 
The loader side should normalize both:
- raw `lora_te_layers_*`
- converted `text_encoders.qwen3_06b.transformer.model.*`
to the actual runtime path:
- `text_encoders.qwen3_06b.model.*`

**Comment by Haoming02** [comment]
Implemented~

**Comment by nArn0** [comment]
> Just be a sane person and don't train the Text Encoder 🫠

Yes indeed, that's what i usually do (and recommend in a guide i made on civitai). But here, i was experimenting with the LoRA structure (and also made a LoRA using SVD between base qwen3-0.6b and the abliterated version). Anyway, thank you so much for adding this 💜 and thanks @plularias for the in depth analysis!
